### PR TITLE
New header

### PR DESCRIPTION
**Pull request by Joozty**

### CODE REVIEW

File: templates/data-providers/index.jsx [hunk 1]
```diff
@@ -1,12 +1,11 @@
-import React, { useEffect, useRef, useLayoutEffect } from 'react'
-import { Button } from '@oacore/design'
+import React, { useEffect, useRef, useLayoutEffect, useCallback } from 'react'
```
Comment (viktor-yakubiv) [inline templates/data-providers/index.jsx:1]: I see you updated React to 17. Are you planning to get rid out of these React import at some point?

File: templates/data-providers/index.jsx [hunk 2]
```diff
@@ -83,9 +82,21 @@ const DataProvidersSearchTemplate = React.memo(
...
     ...formProps
   }) => {
-    useSearchBar({
+    const getSuggestions = useCallback(
+      (term) => {
+        const matches = searchDataProviders(term)
+        return matches.slice(0, 10).map((dataProvider) => ({
+          id: dataProvider.id,
+          value: dataProvider.name,
+          icon: '#magnify',
```
Comment (viktor-yakubiv) [inline templates/data-providers/index.jsx:91]: This icon is not consistent on the `/data-providers` if the popup is not open. If it's not a few minutes fix, I am fine with this inconsistency. We will update this form anyway when the design is ready.

**Review by viktor-yakubiv — APPROVED** [review]
It's awesome 🚀

File: templates/data-providers/index.jsx (hunk 1 above)
Comment (Joozty) [reply templates/data-providers/index.jsx:1]: Yes at some point. There is a bug in nextjs which is preventing us to do it. I cannot find it now but I subscribed to that thread and once it's solved I will do it.

**Pull request merged**